commit cf7387344ca9f78fa5aaaab38f41cf0ab9832568
Author: Christian Kellner <christian@kellner.me>
Date:   Fri Jan 4 11:44:53 2019 +0100

    journal: fix invalid format string in warning
    
    Remove the erroneous extra string format specifier ('%s') that was
    not passed; which of course is a bug and made some architectures
    fall over. Should never actually happen in the wild, because bolt
    tries very hard to not write invalid journal entries ... but who
    knows.

diff --git a/boltd/bolt-journal.c b/boltd/bolt-journal.c
index 715f862..5edd647 100644
--- a/boltd/bolt-journal.c
+++ b/boltd/bolt-journal.c
@@ -490,7 +490,7 @@ bolt_journal_list (BoltJournal *journal,
       if (err != NULL)
         {
           bolt_warn_err (err, LOG_TOPIC ("journal"),
-                         "invalid entry: '%s': %s", l);
+                         "skipping entry '%s'", l);
           continue;
         }
 
